connaisseur/flask_server.py

```diff
@@ -88,16 +87,14 @@ def healthz():
 @APP.route("/ready", methods=["GET", "POST"])
 def readyz():
     """
-    Handles the '/ready' endpoint and checks the health status of the configured notary
-    server and whether the webhook is installed or not. If the notary is available and the
-    webhook is installed it returns a 200 status code. Otherwise should one of them not be
-    reachable, 500 is returned. For installation purposes, the readiness probe first
-    checks whether a specific bootstrap pod (called sentinel) is running in the namespace.
-    If this pod can be found and is still running, the readiness probe automatically
-    returns 200. This bootstrap pod will only run for the first 30 seconds after
+    Handles the '/ready' endpoint. Checks whether the webhook is installed or not.
+    If the notary is available and the webhook is installed it returns a 200 status code.
+    Otherwise should one of them not be reachable, 500 is returned. For installation purposes,
+    the readiness probe also checks whether a specific bootstrap pod (called sentinel) is running
+    in the namespace. If this pod can be found and is still running, the readiness probe automatically
+    returns 200. This bootstrap pod will only run for a short time after
     installation or until the webhook is installed, after which the pod gets immediately
-    deleted. From there on the readiness probe checks the notary server and webhook as
-    usual.
+    deleted. From there on the readiness probe checks the webhook as usual.
```

Comment:
could probably move the `deleted.` to the upper line and make line breaking width consistent.
feel free to drop